lib/superstring/index.js

```diff
@@ -51,7 +51,7 @@ _.assign(SuperString.prototype, /** @lends SuperString.prototype */ {
         }));
 
         this.replacements = replacements; // store the last replacements
-        this.substitutions += Number(replacements); // if any replacement is done, count that some substitution was made
+        replacements && (this.substitutions += 1); // if any replacement is done, count that some substitution was made
```

Comment:
Is the substitutions property being used elsewhere?

Reply:
No.